apps/backend/src/location/location.service.ts

```diff
@@ -31,39 +31,6 @@ export class LocationService {
     return merchant_ids.map((itm) => itm.merchant_id) as number[];
   }
 
-  public async getPTLocationsByID(
```

Comment:
Removed - we can get pt_location_id from the same query used in POS reconciliation